frontend/src/common/featureFlags/features.ts

```diff
@@ -4,3 +4,5 @@ export enum FEATURES {
   GENE_SETS = "gs",
   CURATOR = "rubber-chicken",
 }
+
+export const CURATOR_FEATURES = [FEATURES.CREATE_COLLECTION, FEATURES.AUTH];
```

Comment:
We're planning to deprecate the cc and auth flags, which is why I had them both in code for the moment while Brian communicates the deprecation

Reply:
Ooh I see! I'll add the checks back at callsites 👍